### ISSUE TITLE
iOS: fullscreen doesn't work

### ISSUE THREAD

**Issue by claysquareninety** [codysherman/2nfm#108]
Make it work!

**Comment by mjneff2** [comment]
Looking into this but its somewhat involved actually, as iOS doesn't support Fullscreen API https://stackoverflow.com/questions/43024394/ios10-fullscreen-safari-javascript

**Comment by claysquareninety** [comment]
Yeah I saw that after I created this. Decided to leave it up in case someone wanted to get creative.
